config/initializers/filter_parameter_logging.rb

```diff
@@ -1,6 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [
-  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+Rails.application.config.filter_parameters += %i[
```

Comment:
I don't like this one (though I know a lot of devs do). To me the former is a lot clearer.

Reply:
I am one of them